src/backend/distributed/sql/udfs/citus_disable_node/latest.sql

```diff
@@ -1,9 +1,9 @@
 DROP FUNCTION  pg_catalog.citus_disable_node(nodename text, nodeport integer);
-CREATE FUNCTION pg_catalog.citus_disable_node(nodename text, nodeport integer, force bool default false)
+CREATE FUNCTION pg_catalog.citus_disable_node(nodename text, nodeport integer, force bool default false, sync bool default false)
```

Comment:
does sync stand for metadata syncing or for synchronous?

Reply:
synchronous, changed in all places